build/configs/imxrt1050-evk/imxrt1050-evk_download.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+###########################################################################
+#
+# Copyright 2019 NXP Semiconductors All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+# imxrt1050-evk_download.sh
+
+THIS_PATH=`test -d ${0%/*} && cd ${0%/*}; pwd`
+
+echo $THIS_PATH
```

Comment:
Why do you want to print a location of this script to user?

Reply:
I removed it
Please check it